scripts/pkg/config.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  // Open license ensures JS files are accessible as is
+  // which is important for local templates installation
+  license: 'MIT',
+  files: [
```

Comment:
We might want to add a note somewhere that this should be updated when the `package.json` `files` array is updated

Maybe update the [following section in the RELEASE_CHECKLIST.md](https://github.com/serverless/serverless/blob/master/RELEASE_CHECKLIST.md#prepare-package) document?

Reply:
> We might want to add a note somewhere that this should be updated when the package.json files array is updated

It's not `package.json` :) and package.json `files` is irrelevant to this setup

Reply:
Yes, I know. What I'm saying is that we have a [`files` definition](https://github.com/serverless/serverless/blob/master/package.json#L36-L46) in `package.json` which will be picked up by `npm` whenever we publish a new version and we should keep that definition and this one in sync since both are specifying which pieces should be included in the published artifact.

If we e.g. add a new directory which needs to be included in the binary and the `npm` package we need to touch both files.

Reply:
For binary generation this field (in case of `serverless` package) is irrelevant.

It deducts which files to take by traversing require paths and not by taking everything that's mentioned in pkg.json `files`

Reply:
And for _npm_ `files` property is used as a filter. By default npm will publish whole folder, while `files` instructs it to narrow it to specific directories and files.

Binary packager works differently.